biz.aQute.junit/src/aQute/junit/JUnitEclipseReport.java

```diff
@@ -103,8 +103,8 @@ public void startTest(Test test) {
 		this.current = test;
 		message("%TESTS  ", test);
 
-		BundleContext context = targetBundle.getBundleContext();
 		try {
+			BundleContext context = targetBundle.getBundleContext();
```

Comment:
I have a more robust fix for this I will PR. So you can remove this commit from this PR.

Reply:
See #2925